docs/install/operator/configuring-eventing-cr.md

```diff
@@ -8,11 +8,43 @@ aliases:
 
 The Knative Eventing operator can be configured with these options:
 
+- [Service Configuration by ConfigMap](#service-configuration-by-configMap)
 - [Private repository and private secret](#private-repository-and-private-secrets)
 - [Configuring default broker class](#configuring-default-broker-class)
 
 __NOTE:__ Kubernetes spec level policies cannot be configured using the Knative operators.
 
+## Service Configuration by ConfigMap
```

Comment:
Should this be Eventing Configuration?

Instructions are great, just not sure the title is clear enough that this pertains to eventing?